### PR TITLE
Create issue and pull request templates

### PR DESCRIPTION
**Pull request by charlestian23**
This pull request creates templates for all future issues reports and pull requests. This will allow us to make clearer what we want in issues and pull requests.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<!-- If your pull request is not related to a particular issue, delete the statement below. -->
+Closes #(issue) 
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Enhancement (improvement to an already existing feature)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```
Comment (cjreed121) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]: Line 14 and 25 seem redundant I think. Each PR that requires a docs update should just do it as it's in the checklist.

**Review by Chase-Grajeda — APPROVED** [review]
Templates look good

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# How Has This Been Tested?
+
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. -->
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
```
Comment (cjreed121) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:22]: With the linting PR I think this wouldn't be necessary as the check would just fail if they don't

**Review by cjreed121 — CHANGES_REQUESTED** [review]

**Comment by charlestian23** [comment]
Merged it so we could test out the bug report system, we can still make changes if we feel they're necessary.

**Pull request merged**